File: 30 Days of Code/Day 6. Review Loop/cpp/Day 6. Review Loop.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */ 

    int n;
    string sUserInput;
    string sEvenOut;
    string sOddOut;
    cin >> n;
    cin.ignore();
    for (int i = 0; i < n; i++){
        sUserInput = "";
        sEvenOut = "";
        sOddOut = "";
        
        cin >> sUserInput;
        for (int j = 0; j < sUserInput.size(); j+=2){
            sEvenOut += sUserInput[j];
        }
        for (int j = 1; j < sUserInput.size(); j+=2){
            sOddOut += sUserInput[j];
        }
        cout << sEvenOut << " " << sOddOut << endl;
    }
    return 0;
}

File: 30 Days of Code/Day 6. Review Loop/python/Day 6. Review Loop.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
n = int(input())
for i in range(n):
    user_input = input()
    even_output = "".join([c for i, c in enumerate(user_input) if not i % 2])
    odd_output = "".join([c for i, c in enumerate(user_input) if i % 2])
    print(f"{even_output} {odd_output}")
